### DISCUSSION TITLE
Ajouter le critère "meublé de tourisme" pour les auto-entrepreneur

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2658] [Idées]
Il me semble que celui-ci correspond à un taux de cotisation particulier. 

tag: LMP ; loueur meublé

**Comment by mquandalle** [comment]
À noter les AE meublé de tourisme ne payent pas la nouvelle cotisation d'indemnité journalière maladie https://www.autoentrepreneur.urssaf.fr/portail/accueil/sinformer-sur-le-statut/toutes-les-actualites/creation-dun-dispositif-dindemni.html

**Comment by liliced** [comment]
Obsolète avec https://github.com/betagouv/mon-entreprise/issues/3335
